scripts/mkcloud

```diff
@@ -1338,6 +1340,11 @@ Optional
         If set, does not use the --insecure flag in openstack CLI commands.
     want_monasca_tsdb (Cloud9+ only)
         Allows setting time-series DB used for Monasca [influxdb|cassandra].
+    want_ipv6 (Currently in development)
+        Prepare crowbar to use a full IPv6 single stack control plane. Enabling will also set
+        the firmware_type to uefi as it's required to netboot nodes in IPv6.
+        This mode also depends on SLAAC to know the IP address of the admin node. So radvd
+        or similar will be required to send RA's to let the admin node know how to get/set it's IP.
```

Comment:
Can you drop this bit? I'm getting close to just writing a PR that will configure and start radvd if it isn't running, like Ardana does.

Reply:
Yeah, it is tempting.. cause it's suddenly an new external dependency.